cloudinary-core/src/main/java/com/cloudinary/Url.java

```diff
@@ -417,6 +419,16 @@ public String generate(String source) {
             } catch (MalformedURLException ignored) {
             }
         }
+        if (cloudinary.analytics != null && cloudinary.config.analytics) {
+            try {
+                URL tempUrl = new URL(url);
+                if (tempUrl.getQuery() == null) {
+                    String path = tempUrl.getPath();
+                    url = url + "?" + cloudinary.analytics.toQueryParam();
```

Comment:
what if we have both `authToken` and `analytics`? we need to make sure we build the right query string( using `&`)

Reply:
As part of this ticket: https://cloudinary.atlassian.net/browse/SNI-5996
I needed to prevent the analytics for every URL that has query params, including auth token.

Reply:
Where can I see that in code?
I see this line:
`if (signUrl && authToken != null && !authToken.equals(AuthToken.NULL_AUTH_TOKEN)) {`
and this line:
`if (cloudinary.analytics != null && cloudinary.config.analytics) {`

They both can be evaluated to `true`.